src/style.css

```diff
@@ -0,0 +1,37 @@
+:root {
+  --star-size: 16px;
+  --container-size: 96px;
+}
+
+* {
+  margin: 0;
+}
+
+.container {
+  height: var(--container-size);
+  width: var(--container-size);
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
You should position your stars by using margin between each star, not by using `justify-content: space-between;` This is causing your tests to fail.